plugins/ida/bap_ida_service.ml

```diff
@@ -0,0 +1,154 @@
+open Core_kernel.Std
```

Comment:
did you use `git mv` to migrate this code from `bap_ida.ml` to `bap_ida_service.ml`?